test-it/karma/javascriptinvoke/ReturnDataTypes.Test.js

```diff
@@ -98,6 +97,38 @@ describe('A JavaScript function invoke', function () {
             return 'National Instruments';
         };
 
+        window.NI_Int8ArrayFunction = function () {
+            return Int8Array.from([1, 2, 3]);
+        };
+
+        window.NI_Int16ArrayFunction = function () {
+            return Int16Array.from([1, 2, 3]);
```

Comment:
For integer types, please add the Min and Max value in the integer range. For floating point values please do negative, zero, positive values, at this moment I don't have a better idea about testing them